samples/ComputerVision/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,242 @@
+﻿namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.AnalyzeImage
+{
+    using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+    using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+
+    public static class Program
+    {
+        public const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Services subscription key here
+        public const string endpoint = "https://westus.api.cognitive.microsoft.com"; // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                AnalyzeImageSample.Run(endpoint, subscriptionKey);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+
+        private class AnalyzeImageSample
+        {
+            public static void Run(string endpoint, string subscriptionKey)
```

Comment:
For this, as well as the other cases, please make this async and call them XXXRunAsync. This will also move the .Wait(5000) logic to the Main method where it calls it.